src/components/buttons/button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+/*eslint-disable*/
```

Comment:
- [ ]  It would be better if we fix the ESLint errors instead of disabling the rules. This is a standard for all students of Microverse, it helps us write clean and structured code

Reply:
Done 💯